src/annotator/util/test/buckets-test.js

```diff
@@ -113,163 +113,87 @@ describe('annotator/util/buckets', () => {
-        points.push([200, -1, anchor]);
-      });
-
-      const buckets = buildBuckets(points);
-      assert.equal(buckets.length, 2);
-      assert.isEmpty(buckets[1].anchors);
-      // All anchors are in a single bucket
-      assert.deepEqual(buckets[0].anchors, anchors);
-      // Because this is the first bucket, it will be aligned top
-      assert.equal(buckets[0].position, 150);
+    it('positions the bucket at v. midpoint of the box containing all bucket anchors', () => {
+      fakeAnchors[2].highlights = [200, 50]; // Top 200
+      fakeAnchors[3].highlights = [225, 75]; // Bottom 300
+      const buckets = anchorBuckets(fakeAnchors);
+      assert.equal(buckets[1].position, 250);
     });
 
-    it('should group nearby anchor highlights into shared buckets', () => {
-      let increment = 25;
-      const anchors = [{}, {}, {}, {}];
-      const points = [];
-      // Represents points for 4 anchors that all have different start and
-      // end positions, but only differing by 25px
-      anchors.forEach(anchor => {
-        points.push([150 + increment, 1, anchor]);
-        points.push([200 + increment, -1, anchor]);
-        increment += 25;
-      });
-
-      const buckets = buildBuckets(points);
+    it('only buckets annotations that have highlights', () => {
+      const badAnchor = { highlights: [] };
+      fakeAnchors.push(badAnchor);
+      const buckets = anchorBuckets([badAnchor]);
       assert.equal(buckets.length, 2);
-      assert.isEmpty(buckets[1].anchors);
-      // All anchors are in a single bucket
-      assert.deepEqual(buckets[0].anchors, anchors);
-      // Because this is the first bucket, it will be aligned top
-      assert.equal(buckets[0].position, 175);
+      assert.isEmpty(buckets[0].anchors); // Holder for above-screen anchors
+      assert.isEmpty(buckets[1].anchors); // Holder for below-screen anchors
     });
 
-    it('should put anchors that are not near each other in separate buckets', () => {
-      let position = 100;
-      const anchors = [{}, {}, {}, {}];
-      const points = [];
-      // Represents points for 4 anchors that all have different start and
-      // end positions, but only differing by 25px
-      anchors.forEach(anchor => {
-        points.push([position, 1, anchor]);
-        points.push([position + 20, -1, anchor]);
-        position += 100;
-      });
-      const buckets = buildBuckets(points);
-      assert.equal(buckets.length, 8);
-      // Legacy of previous implementation, shrug?
-      assert.isEmpty(buckets[1].anchors);
-      assert.isEmpty(buckets[3].anchors);
-      assert.isEmpty(buckets[5].anchors);
-      assert.isEmpty(buckets[7].anchors);
-      assert.deepEqual(buckets[0].anchors, [anchors[0]]);
-      assert.deepEqual(buckets[2].anchors, [anchors[1]]);
-      assert.deepEqual(buckets[4].anchors, [anchors[2]]);
-      assert.deepEqual(buckets[6].anchors, [anchors[3]]);
+    it('handles sorting anchors for position', () => {
```

Comment:
```suggestion
    it('sorts anchors by top position', () => {
```

Just a suggestion. I found the previous text slightly awkward.